tests/test_modelgen.py

```diff
@@ -19,9 +55,13 @@ def test_regularization_0size_interval(self):
 
     def test_base_hyper_parameters_reg(self):
         """ Base hyper parameter set should contain regularization. """
-        hyper_parameter_set = modelgen.generate_base_hyper_parameter_set()
+        hyper_parameter_set = modelgen.generate_base_hyper_parameter_set(low_lr=1, 
+                                                                         high_lr=4, 
+                                                                         low_reg=1, 
+                                                                         high_reg=3)
         assert 'regularization_rate' in hyper_parameter_set.keys()
 
+    # Tests for CNN model:
```

Comment:
Better to arrange this in different test suits than with comments. So group the tests in classes.

Reply:
That's a good idea! For now I will still leave it as it is, but hope to soon come back an clean this up!